setup.py

```diff
@@ -246,6 +246,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 databricks = [
     'requests>=2.26.0, <3',
+    'databricks-sql-connector>=1.0.0, <2.0.0',
```

Comment:
This library has a license defect. We can't use it yet.https://github.com/apache/airflow/issues/21030#issuecomment-1019363435
Do you consider using any alternative e.g. databricks-dbapi?

Reply:
That alternative requires an installation of additional components like odbc driver, that isn’t straightforward. I’ll try to clear licensing issue with dev team

Reply:
@mik-laj The source code is downloaded when you install package from pypi.  As it was commented in the original issue, there is no github repo yet, but dev team is working on that.

Reply:
@mik-laj @alexott, 
I have 2 questions,

1. Is it okay to use a library not having code publically available? I personally think, it is not.
2. As Databricks API 2.1 is already available as OpenAPI, How difficult  it's gonna be in the future to upgrade if we don't have opensource repository.

Reply:
1. Code is installed as part of the package, Apache License doesn't require that the code is hosted on github.  Dev team needs some time to create a repo, setup automation, etc.
2. This library isn't related to the API 2.1 at all.